web_external/ImagesGallery/Facets/ImagesFacetView.js

```diff
@@ -14,15 +14,29 @@ isic.views.ImagesFacetView = isic.View.extend({
         this.filter = settings.filter;
 
         this.facetId = this.completeFacet.id;
+        this.facetContentId = this.className + '-' + this.completeFacet.id.replace(/\./g, '-');
         this.title = this.completeFacet.schema().title;
     },
 
     events: {
-        'click .toggle': function (evt) {
-            this.$('.toggle').toggleClass('icon-down-open')
-                .toggleClass('icon-right-open');
+        'show.bs.collapse .isic-images-facet-content': '_toggleCollapseIndicator',
+        'hide.bs.collapse .isic-images-facet-content': '_toggleCollapseIndicator'
+    },
+
+    _toggleCollapseIndicator: function () {
+        this.$('.isic-images-facet-indicator')
+            .toggleClass('icon-down-open')
+            .toggleClass('icon-right-open');
+    },
 
-            this.$('.isic-images-facet-content').toggle();
+    /**
+     * Apply initial collapse state defined in schema.
+     * If 'collapse' is unspecified, default to true.
```

Comment:
We have total control over the schema, so do you want to just always define `collpase` within it? Or do you feel like that's too redundant and burdensome?